client/src/performance/performance.jsx

```diff
@@ -0,0 +1,150 @@
+const performDateFormat = 'MMMM DD, YYYY';
+
+const getStateFromProps = ({ closeAt, id, listExported, name, openAt, performDate }) => ({
+  closeAt: moment(closeAt),
+  editing: false,
+  id,
+  listExported,
+  name,
+  openAt: moment(openAt),
+  performDate: moment(performDate),
+  requesting: false
+});
+
+export default class Performance extends Component {
+
+  static get propTypes() {
+    return {
+      closeAt: PropTypes.string.isRequired,
+      id: PropTypes.number.isRequired,
+      listExported: PropTypes.bool.isRequired,
+      name: PropTypes.string.isRequired,
+      onEdit: PropTypes.func.isRequired,
+      openAt: PropTypes.string.isRequired,
+      performDate: PropTypes.string.isRequired
+    };
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = getStateFromProps(props);
+    this.handleCloseAtChange = this.handleCloseAtChange.bind(this);
+    this.handleConfirm = this.handleConfirm.bind(this);
+    this.handleEditToggle = this.handleEditToggle.bind(this);
+    this.handleNameChange = this.handleNameChange.bind(this);
+    this.handleOpenAtChange = this.handleOpenAtChange.bind(this);
+    this.handlePerformDateChange = this.handlePerformDateChange.bind(this);
+  }
+
+  componentDidReceiveProps(props) {
+    this.setState(getStateFromProps(props));
+  }
+
+  handleCloseAtChange(closeAt) {
+    this.setState({ closeAt });
+  }
+
+  handleConfirm() {
+    const { state } = this;
+
+    delete state.editing;
+    delete state.requesting;
+
+    this.props.onEdit({
+      ...state,
+      closeAt: state.closeAt.format(),
+      openAt: state.openAt.format(),
+      performDate: state.performDate.format()
+    });
+  }
+
+  handleEditToggle() {
+    this.setState(({ editing }) => ({ editing: !editing }));
+  }
+
+  handleNameChange({ target }) {
+    this.setState({ [target.name]: target.value });
+  }
+
+  handleOpenAtChange(openAt) {
+    this.setState({ openAt });
+  }
+
+  handlePerformDateChange(performDate) {
+    this.setState({ performDate });
+  }
+
+  render() {
+    const { editing, requesting } = this.state;
+    const { closeAt, name, openAt, performDate } = this.state;
+    const listExported = false;
+
+    return (
+      <Card>
+        <CardHeader
+          title={editing ?
+            <div className="Performance-dateWrapper">
+              <label>Performance Name:&nbsp;</label>
+              <input onChange={this.handleNameChange} name="name" value={name} />
+            </div> :
+            name
+          }
+          subtitle={editing ?
+            <div className="Performance-dateWrapper">
+              <label>Performance Date:&nbsp;</label>
+              <Datetime
+                onChange={this.handlePerformDateChange}
+                textFormat={performDateFormat}
+                value={performDate}
+              />
+            </div> :
+            `Performance on ${moment(performDate).format(performDateFormat)}`
+          }
+        />
+        {editing ?
```

Comment:
We should be checking for `listExported` to change the open and close times. If the list is exported, a performance shouldn't be exported